backend/census_historical_migration/workbooklib/census_models/census.py

```diff
@@ -14,7 +14,7 @@ def model_module_path(model, year):
 
 
```

Comment:
Do we still need census_models? The models are already present in models.py. 
Besides, we want to avoid dynamic imports if we can.

Reply:
Not needed but it can't be removed as part of this PR.